backend/BackendMIIT/src/main/java/com/example/BackendMIIT/service/DirectionService.java

```diff
@@ -15,6 +17,8 @@ public interface DirectionService {
 
     DirectionDto getDirectionByCode(String code);
 
-
     List<DirectionDto> getDirections();
+
+    Page<DirectionWithProfilesDto> getSortedDirections(String typeOfPassPoints, int page, int size);
```

Comment:
Rename variable typeOfPassPoints. To rename, use shift + f6. It allows you to rename variable and its usages